### PR TITLE
Update README + announcement guidelines

### PR DESCRIPTION
**Pull request by alxs**

### CODE REVIEW

**Review by Noc2 — APPROVED** [review]

File: grants/announcement-guidelines.md [hunk 1]
```diff
@@ -2,61 +2,69 @@
...
+## We recommend the following guidelines when writing your post
 
 - The post should be strategic in nature, focusing on the tech rather than the “announcement” element.
 - The team should point to the work, in order to attract attention for the project and demonstrate momentum in the ecosystem. It should be informative for builders and the larger community.
-- The post should include a link to github repo or elsewhere, to showcase what’s built so far.
+- The post should include a link to a GitHub repository or elsewhere to showcase what's been built so far.
 - Suggested flow for the post:
   - This is what the team has built so far.
-  - This is what the team's plan is for continued development.
+  - These are the team's plans to continue development.
```
Comment (semuelle) [inline grants/announcement-guidelines.md:21]: Weird phrasing.

```suggestion
  - These are the team's development plans.
```

File: grants/announcement-guidelines.md [hunk 2]
```diff
@@ -2,61 +2,69 @@
...
   - This is how the technology will contribute to the Polkadot ecosystem.
 
 ## Key components
 
 1. A blog post header image
-  - An image helps when sharing the announcement on social. It’s more prominent in the social feeds and looks more professional.
-  - Images ideas include showing how W3F or Polkadot fits into your flow. Or you can simply put the teams’ logos next to each other.
+
+   - An image helps when sharing the announcement on social networks. It's more prominent in feeds and looks more professional.
+   - Images ideas include showing how W3F or Polkadot fits into your flow. Or you can simply put the teams' logos next to each other.
```
Comment (semuelle) [inline grants/announcement-guidelines.md:29]:
```suggestion
   - Image ideas include showing how W3F or Polkadot fit into your flow. Or you can simply put the teams' logos next to each other.
```

File: grants/grant_application_template.md [hunk 3]
```diff
@@ -2,72 +2,86 @@
...
+Below we provide an **example roadmap**. In the descriptions it should be clear how the project is related to Substrate and/or Polkadot. We recommend that the scope of the work can fit within a 3 month period and that teams structure their roadmap as 1 month = 1 milestone.
 
 For each milestone:
+
 * Please be sure to include a specification of the software. The level of detail must be enough so that we are able to verify that the software meets the specification.
 * Please include total amount of funding requested per milestone.
 * Please note that we require documentation (e.g. tutorials, API specifications, architecture details) in each milestone. This ensures that the code can be widely used by the community.
 * Please provide a test suite, comprising unit and integration tests, along with a guide on how to run these.
-* Please commit to providing a dockerfiles for the delivery of your project. 
+* Please commit to providing a dockerfiles for the delivery of your project.
```
Comment (semuelle) [inline grants/grant_application_template.md:74]:
```suggestion
* Please commit to providing a `Dockerfile` for the delivery of your project.
```

**Review by semuelle — COMMENTED** [review]
Nice. Sorry to nitpick, but could you replace all the right-leaning quotation marks with regular ones? 😬

**Comment by alxs** [comment]
> Nice. Sorry to nitpick, but could you replace all the right-leaning quotation marks with regular ones?

Nothing wrong with those, no? :)

**Review by semuelle — APPROVED** [review]

**Pull request merged**